test/sbd_config

```diff
@@ -0,0 +1,115 @@
+## Type: string
```

Comment:
I'm guessing these are here for testing purposes? If so, I'd strip all comments from the original file and add some brief ones explaining what use case this config is testing

Reply:
Hi @rtorrero ,
I have decided the let the test files as they are. Using exact real use cases help to find strange errors.
About the description, I think that the file names are enough meaningful